plugins/flytekit-duckdb/setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import setup
+
+PLUGIN_NAME = "duckdb"
+
+microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
+
+plugin_requires = ["flytekit>=1.3.0b2,<2.0.0", "duckdb"]
+
+__version__ = "0.0.0+develop"
+
+setup(
+    name=microlib_name,
+    version=__version__,
+    author="flyteorg",
+    author_email="admin@flyte.org",
+    description="DuckDB Plugin for Flytekit",
+    namespace_packages=["flytekitplugins"],
+    packages=[f"flytekitplugins.{PLUGIN_NAME}"],
+    install_requires=plugin_requires,
+    license="apache2",
+    python_requires=">=3.7",
```

Comment:
let's restrict this to <=3.10 as well, [like this](https://github.com/flyteorg/flytekit/blob/master/setup.py#L84)